src/Hl7.Fhir.ElementModel.Shared.Tests/ScopedNodeTests.cs

```diff
@@ -1,344 +1,344 @@
-            Assert.IsNotNull(typedElements.First().Value);
-        }
-        
-        [TestMethod]
-        public void Bundle_WithEntryWithoutFullUrl_ShouldNotThrow()
-        {
-            var bundle = new Bundle() { Type = Bundle.BundleType.Batch, Entry = [new Bundle.EntryComponent() { Resource = new Patient() }]};
-
-            var enumerate = () => bundle.ToPocoNode().BundledResources();
-            enumerate.Should().NotThrow().Subject.Should().ContainSingle(c => !c.Children("fullUrl").Any());
-        }
-
-        private class CCDAResourceResolver : IAsyncResourceResolver
-        {
-            private readonly Dictionary<string, StructureDefinition> _cache;
-            private readonly CachedResolver _coreResolver;
-
-            public CCDAResourceResolver()
-            {
-                _cache = new Dictionary<string, StructureDefinition>();
-                _coreResolver = new CachedResolver(new MultiResolver(
-                    ZipSource.CreateValidationSource(),
-                    new DirectorySource("TestData/TestSd")));
-            }
-
-            public async Tasks.Task<Resource> ResolveByCanonicalUriAsync(string uri)
-            {
-                if (_cache.TryGetValue(uri, out StructureDefinition? sd))
-                    return sd;
-
-                sd = await _coreResolver.FindStructureDefinitionAsync(uri);
-                if (!sd.HasSnapshot)
-                {
-                    var snapShotGenerator = new SnapshotGenerator(_coreResolver);
-                    await snapShotGenerator.UpdateAsync(sd);
-                    _cache.Add(sd.Url, sd);
-                }
-
-                return sd;
-            }
-
-            public Tasks.Task<Resource> ResolveByUriAsync(string uri) => throw new NotImplementedException();
-        }
-
-        private class TypedElementWithoutDefinition : ITypedElement, IResourceTypeSupplier
-        {
-            private readonly ITypedElement _wrapped;
-
-            public TypedElementWithoutDefinition(ITypedElement wrapped) => _wrapped = wrapped;
-
-            public string Name => _wrapped.Name;
-
-            public string? InstanceType => _wrapped.InstanceType;
-
-            public object? Value => _wrapped.Value;
-
-            public string Location => _wrapped.Location;
-
-            public IElementDefinitionSummary? Definition => null;
-
-            public string? ResourceType => !string.IsNullOrEmpty(InstanceType) && char.IsUpper(InstanceType, 0) ? InstanceType : null;
-
-            public IEnumerable<ITypedElement> Children(string? name = null) =>
-                _wrapped.Children(name).Select(c => new TypedElementWithoutDefinition(c));
-        }
-
-        private class NoTypeProvider : IStructureDefinitionSummaryProvider
-        {
-            public IStructureDefinitionSummary? Provide(string canonical) => null;
-        }
-    }
-}
\ No newline at end of file
+﻿// using FluentAssertions;
```

Comment:
Should these checks be re enabled to test the new extension methods that do the same thing?